README.md

```diff
@@ -1 +1,8 @@
-# cli
\ No newline at end of file
+# Aspen Mesh CLI
+These are the official scripts/CLI for [Aspen Mesh](https://aspenmesh.io/).
+
+## Aspen Mesh Collector Installation
+The Aspen Mesh Collector can be installed by a one-line script:
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/aspenmesh/cli/main/install.sh)" -- --api-key <initial-signup-api-key>
```

Comment:
`install.sh` is the suggested script name by Rick. Let's see if we should name it more specifically. Perhaps we should reconsider it when we have more scripts?

Reply:
That seems okay; since a user will likely copy/paste/run this out of the UI, if we change the name, we can change the UI (i.e. I don't think backwards-compatibility would matter in terms of script naming)